packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -109,6 +121,15 @@ export const CommandMenu = () => {
     [toggleCommandMenu, setSearch],
   );
 
+  useScopedHotkeys(
+    'esc,Esc,Escape,escape',
```

Comment:
Think you just need `'esc'` here. With `'esc,Esc,Escape,escape'`, the `closeCommandMenu` is called 4 times

Reply:
Ok, will try that too.